.github/workflows/torchserve-nightly-build.yml

```diff
@@ -6,28 +6,37 @@ on:
     - cron:  '15 11 * * *'
 jobs:
   nightly:
-    runs-on: ubuntu-18.04
+    # creates workflows for the 3 OS: ubuntu, macOS & windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
-      - name: Setup Python 3.8
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-          architecture: x64
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1
+      - run: conda --version
+      - run: conda install -y conda-build anaconda-client
       - name: Checkout TorchServe
         uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          set -eux
-          pip install twine
-          sudo apt-get update
```

Comment:
Now, twine will be installed when user runs
python ts_scripts/install_dependencies.py --environment=dev

twine is included in developer.txt

Changed this based on request by @msaroufim

Reply:
twine is 36KB, it's not the reason why our images will get larger. It's not good practice to `os.system(pip install)` in a script because you will change user machine without them being aware. Either we add it in dev dependencies or we run it as a step in Github action and include it in documentation in README.